hades/__init__.py

```diff
@@ -219,23 +220,20 @@ def submit():
         user.email += f", {request.form['email_second_person']}"
         user.department += f", {request.form['department_second_person']}"
 
-    # Ensure that no data is duplicated. If anything is wrong, display the corresponding error to the user
```

Comment:
Ideally this should be implemented - maybe in the form of a separate API we can call with AJAX before actual form submission, because the MongoDB validation runs on insert, so we can't really extract an exact error message to display, with this we can check before insert and display the exact issue to the user